### PR TITLE
Change storage solution from bucketeer to gcp

### PR DESCRIPTION
**Pull request by ycaokris**
Use interpolated key to get rid of heroku storage solution, direct storage bucket to gcp.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -533,4 +542,4 @@ def reset_dropdown_enhance(_):
 
 # Running the server
 if __name__ == '__main__':
-    app.run_server(debug=True)
+    app.run_server(debug=True, port=8051)
```
Comment (nicolaskruchten) [inline app.py]: why is this required? Is this a DDS thing?

**Comment by nicolaskruchten** [comment]
Seems fine to me, if it breaks the Bucketeer dependency for the gallery-deployed version :)

**Comment by nicolaskruchten** [comment]
💃

File: app.py (hunk 1 above)
Comment (ycaokris) [reply app.py]: A port setting for personal debugging. Will remove it.
